python/dgl/graphbolt/feature_store.py

```diff
@@ -142,21 +142,29 @@ def update(self, value: torch.Tensor, ids: torch.Tensor = None):
             self._tensor[ids] = value
 
 
-# FIXME(Rui): To avoid circular import, we make a copy of `OnDiskDataFormatEnum`
-# from dataset.py. We need to merge the two definitions later.
-class OnDiskDataFormatEnum(pydantic_yaml.YamlStrEnum):
-    """Enum of data format."""
+class FeatureDataFormatEnum(pydantic_yaml.YamlStrEnum):
```

Comment:
Merge the whole implementation here to python/dgl/graphbolt/impl/ondisk_dataset.py.

Reply:
`OnDiskDataset` should use the class defined here as `feature_store` is a dependency of `OnDiskDataset`.

Reply:
There are several APIs defined in graphbot/, and all the implementation should go into graphbot/impl. We can put all OnDiskRelated loading code to a single python file, and it depends on related API defined in graphbot/.

Reply:
sure. I added a `TODO` comment ahead this line and I'd like to finish it in another separate PR.